msgcntr/messageforums-app/src/webapp/jsp/discussionForum/message/dfAllMessages.jsp

```diff
@@ -14,15 +14,16 @@
 
 <!-- messageforums-app/src/webapp/jsp/discussionForum/message-->
 
-<script type="text/javascript">includeLatestJQuery("msgcntr");</script>
```

Comment:
Are all of these changes related to the PR?

Reply:
I took the type attribute off the js scripts. Is that what you mean? They're not needed and I like to clean them up as I go.